css/ampdoc.css

```diff
@@ -774,61 +293,14 @@ amp-experiment, amp-share-tracking {
   padding: 5px;
 }
 
-/**
- * amp-accordion to avoid FOUC.
- */
-
-/* Non-overridable properties */
-amp-accordion {
-  display: block !important;
-}
-
-/* Make sections non-floatable */
-amp-accordion > section {
-  float: none !important;
-}
-
-/*  Display the first 2 elements (heading and content) */
-amp-accordion > section > * {
-  float: none !important;
-  display: block !important;
-  overflow: hidden !important; /* clearfix */
-  position: relative !important;
-}
-
-amp-accordion,
-amp-accordion > section,
-.i-amphtml-accordion-header,
-.i-amphtml-accordion-content {
-  margin: 0;
-}
-
-/* heading element*/
-.i-amphtml-accordion-header {
-  cursor: pointer;
-  background-color: #efefef;
-  padding-right: 20px;
-  border: solid 1px #dfdfdf;
-}
-
-/* Collapse content by default. */
-amp-accordion > section > :last-child {
-  display: none !important;
-}
-
-/* Expand content when needed. */
-amp-accordion > section[expanded] > :last-child {
-  display: block !important;
-}
-
 /* amp-list CSS to avoid FOUC */
 
 /**
  * The default amp-hidden changes visibility, which takes up space. In the case
  * where amp-list becomes a container via the resizable-children attribute, the
  * loader should be fully hidden and not take up space.
  */
- amp-list[resizable-children] > .i-amphtml-loading-container.amp-hidden {
```

Comment:
Why are comments above staying here?

Reply:
The above comments are about amp-list, which is not allowed in ads.

Reply:
Even though it's not allowed in ads, I still think it should be in shared/element css. It has no real affinity to the document-level css.